tests/cases/fourslash/completionsImport_multipleWithSameName.ts

```diff
@@ -21,7 +17,6 @@ verify.completions({
     marker: "",
     exact: [
         "globalThis",
-        { name: "foo", text: "var foo: number", kind: "var", kindModifiers: "declare" },
```

Comment:
The global foo should still be shown? since c.ts is global file and not external module (yet)

Reply:
I removed the global `foo` above because this PR reverts the decision documented here:

> A local variable would prevent import completions (see `completionsImport_shadowedByLocal.ts`), but a global doesn't.

If I kept the file `/global.d.ts`, you're right, it would still show up, but the imports below would _not_ show up, and this test would be testing the wrong thing.